xen/arch/arm/coproc/coproc.c

```diff
@@ -360,13 +360,16 @@ struct coproc_device *coproc_alloc(struct dt_device_node *np,
             goto out;
         }
 
-        mmio->base = ioremap_nocache(mmio->addr, mmio->size);
-        if ( !mmio->base )
+        if ( !(coproc->driver_features & COPROC_DRIVER_NO_MMIO_MAP) )
```

Comment:
...SELF_MMIO_MAP ?

Reply:
...would mean we will map. but we might not, so we just ask the framework not to map for us